server/data/incidentClient.ts

```diff
@@ -95,12 +95,7 @@ export default class IncidentClient {
           where r.status = $1
           and   r.agency_id = $2
           order by r.incident_date`,
-      values: [
-        ReportStatus.SUBMITTED.value,
-        agencyId,
-        StatementStatus.PENDING.value,
-        StatementStatus.REMOVAL_REQUESTED.value,
-      ],
+      values: [ReportStatus.SUBMITTED.value, agencyId, StatementStatus.PENDING.value],
```

Comment:
Just wondering, would the `StatementStatus.PENDING.value` be accessed here is there are only 2 attribute placeholders for each status and agency_id?

Reply:
as discussed